mirobo/strip.py

```diff
@@ -0,0 +1,62 @@
+from .device import Device
+from typing import Any, Dict
+
+
+class Strip(Device):
+    """Main class representing the smart strip."""
+
+    def on(self):
+        """Power on."""
+        return self.send("set_power", ["on"])
+
+    def off(self):
+        """Power off."""
+        return self.send("set_power", ["off"])
+
+    def status(self):
+        """Retrieve properties."""
+        properties = ['power', 'temperature', 'current', 'mode']
```

Comment:
Do you have an example response? It would be nice to be added either here or into the Status class.